### PR TITLE
raft: add tag for panic messages

### PR DESCRIPTION
**Pull request by hhkbp2**
For issue #1084

### CODE REVIEW

File: src/raft/raft_log.rs [hunk 1]
```diff
@@ -53,22 +53,25 @@ pub struct RaftLog<T: Storage> {
...
+
+    pub tag: String,
 }
 
 impl<T> ToString for RaftLog<T>
     where T: Storage
 {
     fn to_string(&self) -> String {
-        format!("committed={}, applied={}, unstable.offset={}, unstable.entries.len()={}",
+        format!("committed={}, applied={}, unstable.offset={}, unstable.entries.len()={}, tag={}",
```
Comment (siddontang) [inline src/raft/raft_log.rs]: need tag here?

Comment (siddontang) [inline src/raft/raft_log.rs]: I think we can add tag here https://github.com/pingcap/tikv/blob/e38392f3fe82c1b3721aefdb385d42a9d16cef97/src/raft/raft_log.rs#L398

**Comment by siddontang** [comment]
Rest LGTM
PTAL @BusyJay

File: src/raft/raft_log.rs (hunk 1 above)
Comment (hhkbp2) [inline src/raft/raft_log.rs]: Address comment.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**